web-service/helpers.py

```diff
@@ -189,12 +196,8 @@ def GetContentEncoding(content):
             encoding = value[0]
```

Comment:
Therefore, I think we should try to decode with the encoding passed as parameter and invalidate it if it fails. The fallback on iso-8859-1 should always work.